### DISCUSSION TITLE
Duplicate Draft to Another Draft (or Entry)

### DISCUSSION THREAD

**Discussion by RitterKnightCreative** [craftcms/cms#8763] [Ideas]
### Description
It would be handy to be able to duplicate an existing draft you're working on to either the same entry or a brand new entry.

Say a client or colleague asks about how to do something. Oftentimes you don't want to change their in-progress work. Sometimes you can get away with creating a new draft from the "live" entry, then making the changes necessary. However, if someone's  made some major changes to the content, it becomes much harder to make those edits to "reflect" what they're working on.

Also, when working on an existing draft, sometimes you realize the new content you're creating should actually be a brand new entry instead. Being able to splinter off that content into a new entry would be handy.

In both cases, one workaround is to make the current draft go live so you can make a draft of it then revert the draft back to a previous version, however that seems pretty clumsy. It also has the side effect of blowing away the Draft a client may bookmarked or have named.

These tools already existing for the current version of an entry, please bring them to drafts as well.

**Comment by green17** [comment]
This would be very handy.
